syntaxhighlighter.php

```diff
@@ -337,6 +337,14 @@ function enqueue_block_editor_assets() {
 				: $this->pluginver
 		);
 
+		wp_enqueue_style(
+			'syntaxhighlighter-blocks-css', // Handle.
+			plugins_url( 'dist/blocks.editor.build.css', __FILE__ ),
+			( ( defined( 'WP_DEBUG' ) && WP_DEBUG ) || ( defined( 'SCRIPT_DEBUG' ) && SCRIPT_DEBUG ) )
+				? filemtime( plugin_dir_path( __FILE__ ) . 'dist/blocks.build.js' )
```

Comment:
Although it would probably be the same, I wonder this should reference the css file instead. Otherwise, we could just move this to a shared variable for asset version.

Reply:
Fixed in 55ef01e